src/scripts/config/config_test.go

```diff
@@ -0,0 +1,200 @@
+package config
+
+import (
+	"os"
+	"testing"
+)
+
+func TestExpandEnvVariables(t *testing.T) {
+	tests := []struct {
+		configVar   string
+		envVars     map[string]string
+		expectedErr string
+		expectedVal string
+		fieldName   string
+		id          string
+	}{
+		{
+			// This test case checks the basic functionality of variable expansion.
+			// An environment variable "TEST_VAR" is set, and it is expected that
+			// the variable in the configuration gets expanded to the value of "TEST_VAR".
+			configVar:   "${TEST_VAR}",
+			envVars:     map[string]string{"TEST_VAR": "value"},
+			expectedErr: "",
+			expectedVal: "value",
+			fieldName:   "AccessToken",
+			id:          "1",
```

Comment:
~ rather than a numeric id, I personally prefer to have a description of the test case. It prevents tool-assisted runs outputting confusing results
![image](https://github.com/CircleCI-Public/slack-orb-go/assets/706204/c655a256-119f-4e4b-983a-5324c889f5b8)